docs/incident_response.md

```diff
@@ -0,0 +1,106 @@
+# MozMEAO incident response guide
+
+Stay cool, take a deep breath. We'll solve this problem together, and this guide is here to help.
+
+This document does not describe the on-call or escalation process, but rather the process we use once an issue has been identified.
+
+## Declaring an incident and the incident leader (IL) role
+
+The incident leader (IL) serves as the primary point of contact and coordinator for an incident. This person may have been called by the moc, or just noticed that a service is broken. 
+
+When an incident occurs, the IL posts messages in IRC stating:
```

Comment:
Which IRC channel is this information posted to?

Reply:
I was thinking that a message should be posted in the primary channel where backend engineers idle for each app:
- #mdndev
- #www
- #basket
- #snippets
- #sumo
etc.